cvat-core/src/axios-tus.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import Axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import * as tus from 'tus-js-client';
+
+class AxiosHttpResponse implements tus.HttpResponse {
+    readonly #axiosResponse: AxiosResponse;
+
+    constructor(axiosResponse: AxiosResponse) {
+        this.#axiosResponse = axiosResponse;
+    }
+
+    getStatus(): number { return this.#axiosResponse.status; }
+    getHeader(header: string): string { return this.#axiosResponse.headers[header.toLowerCase()]; }
```

Comment:
Probably it also may return `undefined` as in `AxiosHttpRequest`

```suggestion
    getHeader(header: string): string | undefined { return this.#axiosResponse.headers[header.toLowerCase()]; }
```

Reply:
Good point, fixed. This is actually a bug in the interface definition, so I also filed tus/tus-js-client#706.